### DISCUSSION TITLE
How I can install keda on AKS with a windows node

### DISCUSSION THREAD

**Discussion by rag70** [kedacore/keda#1987] [General]
I need to install "kedo" in an AKS that contains a windows node. When I install it, it is installed on the windows node and not on the Linux one. I have already tried various ways like

````
helm install keda kedacore/keda --namespace keda `
    --set nodeSelector:"beta\.kubernetes\.io\/os"=linux 
````

![image](https://user-images.githubusercontent.com/40672014/127381138-90dc3d2f-4d80-4fc5-92d2-24c476d5c446.png)

**Comment by zroubalik** [comment]
This is more like helm related question. You should check that the labels on those linux nodes exists. And then double check the helm install syntax

**Reply by zroubalik** [reply]
```
helm install keda kedacore/keda --namespace keda --set nodeSelector."beta\.kubernetes\.io\/os"=linux 
```
